dd-sdk-android/src/main/kotlin/com/datadog/android/DatadogConfig.kt

```diff
@@ -111,7 +111,8 @@ private constructor(
                     gesturesTracker = it.gesturesTracker,
                     userActionTrackingStrategy = it.userActionTrackingStrategy,
                     viewTrackingStrategy = it.viewTrackingStrategy,
-                    rumEventMapper = it.rumEventMapper
+                    rumEventMapper = it.rumEventMapper,
+                    longTaskTrackingStrategy = null
```

Comment:
We should keep the same pattern as for the others strategies here

Reply:
Also I don't see where you actually added the method to update this value in the Config Builder.

Reply:
I only added this in the `Configuration` class, not this one.